packages/core/src/shared/lsp/manifestResolver.ts

```diff
@@ -64,9 +64,17 @@ export class ManifestResolver {
         }).getNewETagContent(this.getEtag())
 
         if (!resp.content) {
-            throw new ToolkitError(
-                `New content was not downloaded; fallback to the locally stored "${this.lsName}" manifest`
-            )
+            const msg = `"${this.lsName}" manifest unchanged, skipping download: ${this.manifestURL}`
```

Comment:
`!resp.content` means "etag didnt change". It doesn't mean "failed to download".